src/main/kotlin/translations/lang/spellcheck/LangSpellcheckingStrategy.kt

```diff
@@ -0,0 +1,36 @@
+/*
+ * Minecraft Dev for IntelliJ
+ *
+ * https://minecraftdev.org
+ *
+ * Copyright (c) 2020 minecraft-dev
+ *
+ * MIT License
+ */
+
+package com.demonwav.mcdev.translations.lang.spellcheck
+
+import com.demonwav.mcdev.translations.lang.MCLangLanguage
+import com.demonwav.mcdev.translations.lang.gen.psi.LangTypes
+import com.intellij.psi.PsiElement
+import com.intellij.psi.impl.source.tree.LeafPsiElement
+import com.intellij.spellchecker.tokenizer.SpellcheckingStrategy
+import com.intellij.spellchecker.tokenizer.Tokenizer
+import com.intellij.spellchecker.tokenizer.TokenizerBase
+
+class LangSpellcheckingStrategy : SpellcheckingStrategy() {
+    private val langCommentTokenizer = LangCommentTokenizer()
+    private val langKeyTokenizer = TokenizerBase<LeafPsiElement>(LangKeySplitter)
+
+    override fun isMyContext(element: PsiElement) = MCLangLanguage.`is`(element.language)
+
+    override fun getTokenizer(element: PsiElement?): Tokenizer<*> {
```

Comment:
`element` is never null either from what I saw in the method usages.

Reply:
I kept the nullability marker here because it could be possible that a null element is passed in depending on the PSI tree and file structure of the lang file, along with it being possible to change in the future. Right now, this shouldn't be possible but I still kept it.